interface/patient_file/encounter/forms.php

```diff
@@ -26,10 +34,13 @@
 use OpenEMR\Common\Twig\TwigContainer;
 use OpenEMR\Core\Header;
 use OpenEMR\Events\Encounter\EncounterMenuEvent;
+use OpenEMR\Events\Encounter\EncounterFormsListRenderEvent;
+use OpenEMR\Common\Forms\FormLocator;
+use OpenEMR\Common\Forms\FormReportRenderer;
```

Comment:
keep use alphabetized here and in other scripts

Reply:
Fixed.